tox.ini

```diff
@@ -1,8 +1,8 @@
 [tox]
-envlist = tests,import-minimal
+envlist = import-minimal,tests
 isolated_build = true
 
-[testenv]
+[testenv:tests]
```

Comment:
Not sure what you are trying to do here? Testing only with a single (unspecified) Python version now?

Reply:
Yes, the matrix was not being used, I was only running a single python version. It is now specified in the gh workflow. Do you think we should test all versions?

Reply:
> Also, I think you need a base env (testenv), probably?

Not sure I understand that part.

Reply:
> Do you think we should test all versions?

Not necessarily, but:

- We must test with the lowest version that Scipp supports, which is 3.8, *not* 3.9.
- Would be nice to be specified in the `tox.ini` so it is visible also when running locally, without GH.

>> Also, I think you need a base env (testenv), probably?
>
> Not sure I understand that part.

Maybe it works without in this case, but `testenv` also serves as a base for the other `testenv:abc` environments (they inherite, but can override, settings).